spec/system/claims/schools/users/copy_view_users_spec.rb

```diff
@@ -1,68 +1,68 @@
+    then_i_see_the_claim_users_page
     and_i_select_add_user
     then_i_fill_in_the_new_user_details
     and_i_select_continue
-    then_i_see_the_confirm_user_page
+    then_i_see_the_confirm_user_details_page
     and_i_select_confirm_and_add_user
-    then_i_see_the_new_user_ordered_in_the_users_list
+    then_i_see_the_claim_users_page_with_new_user
 
     when_i_select_the_new_user
-    then_i_see_their_details
+    then_i_see_the_details_page_for_the_new_user
   end
 
   private
 
   def given_a_school_exists_with_claims_users
-    @user_anne = create(:claims_user, :anne)
-    @user_charles = create(:claims_user, first_name: "Charles")
-    @user_bobby_g = create(:claims_user, first_name: "Bobby", last_name: "G")
-    @user_bobby_a = create(:claims_user, first_name: "Bobby", last_name: "A")
+    @user_anne = create(:claims_user, first_name: "Anne", last_name: "Wilson", email: "anne_wilson@education.gov.uk")
+    @user_charles = create(:claims_user, first_name: "Charles", last_name: "G", email: "charles_g@education.gov.uk")
+    @user_bobby_g = create(:claims_user, first_name: "Bobby", last_name: "G", email: "bobby_g@education.gov.uk")
+    @user_bobby_a = create(:claims_user, first_name: "Bobby", last_name: "A", email: "bobby_a@education.gov.uk")
 
     @shelbyville_school = create(
       :claims_school,
-      urn: "123456",
       name: "Shelbyville Elementary",
-      address1: "Westgate Street",
-      address2: "Hackney",
-      postcode: "E8 3RL",
-      group: "Local authority maintained schools",
-      phase: "Primary",
-      gender: "Mixed",
-      minimum_age: 3,
-      maximum_age: 11,
-      religious_character: "Does not apply",
-      admissions_policy: "Not applicable",
-      urban_or_rural: "(England/Wales) Urban major conurbation",
-      percentage_free_school_meals: 15,
-      rating: "Outstanding",
+      id: "a1bda3e6-0e75-45f2-bd77-7d5314afc2a2",
       users: [@user_anne, @user_charles, @user_bobby_g, @user_bobby_a],
     )
   end
 
-  def and_i_am_signed_in_as_a_lead_mentor_user
-    user_exists_in_dfe_sign_in(user: @user_anne)
-    visit sign_in_path
-    click_on "Sign in using DfE Sign In"
+  def and_i_am_signed_in_as_a_user
+    sign_in_claims_user(organisations: [@shelbyville_school])
   end
 
   def when_i_navigate_to_the_claim_school_users_page
     visit claims_school_users_path(@shelbyville_school)
   end
 
-  def and_i_see_the_claim_page_containing_a_list_of_users_ordered_by_full_name
+  def then_i_see_the_claim_users_page
     expect(page).to have_title("Claim funding for mentor training - GOV.UK")
-    expect(page.body.index("Anne")).to be < page.body.index("Bobby A")
-    expect(page.body.index("Bobby A")).to be < page.body.index("Bobby G")
-    expect(page.body.index("Bobby G")).to be < page.body.index("Charles")
+    expect(primary_navigation).to have_current_item("Users")
+    expect(page).to have_h1("Users", class: "govuk-heading-l")
```

Comment:
```suggestion
    expect(page).to have_h1("Users")
```
For the sake of your sanity, I think it's okay to not test that the class is specifically `govuk-heading-l`, the matcher already specifics that it must be `govuk-heading-` and that it's a `h1` which I think is good enough for the purposes of this spec.

https://github.com/DFE-Digital/itt-mentor-services/blob/33810ebfb8ce347df00383a071eb8fb3bf37d766/spec/support/govuk_component_matchers.rb#L82-L89

Reply:
Yeah that makes a lot of sense, I've got rid of them